modules/README.md

```diff
@@ -2,43 +2,82 @@
 
-
 # Writing a package of your own
+
 If you wish to write your own package you need to handle two types of request that are specified as command-line arguments to your program.
 
 ## Manifest
-When your program is called with the arguments `$ ./my_program manifest` it expects to recieve a json object (sent via stdout) that explains how your package works and what transforms it supports. A manifest might look something like this:
+
+When your program is called with the arguments `$ ./my_program manifest` it expects to receive a json object (sent via stdout) that explains how your package works and what transforms it supports. A manifest might look something like this:
+
 ```json
 {
     "name": "Name of your package",
     "version": "0.1",
     "description": "A short description that explains the function of your package",
     "transforms": [
-            {
-                "from": "foo",
-                "to": ["html", "tex"],
-                "arguments": [
-                    {"name": "a_argument", "description": "This is the description for the argument 'a_argument'"},
-                    {"name": "another_argument", "default": "hello", "description": "..."}
-                ],
-            },
-            {
-                "from": "bar",
-                "arguments": [],
-            }
+        {
+            "from": "foo",
+            "to": [
+                "html",
+                "tex"
+            ],
+            "arguments": [
+                {
+                    "name": "a_argument",
+                    "description": "This is the description for the argument 'a_argument'"
+                },
+                {
+                    "name": "another_argument",
+                    "default": "hello",
+                    "description": "..."
+                }
+            ]
+        },
+        {
+            "from": "bar",
+            "arguments": []
+        }
     ]
 }
 ```
 
 ## Transforming an element
-When the program is called with `$ ./my_program transform <element_name> <output_format>` you will need to transform an element into the youdesired output format.
 
-You will be sent the element as json object via stdout and you respond by sending a json list of objects representing elements back.
+When the program is called with `$ ./my_program transform <element_name> <output_format>` you will need to transform an element into the desired output format.
+
+You will be sent the element as json object via stdout, and you respond by sending a json list of objects representing elements back.
+
+If you want to see some examples of what this json format looks like, you can check out the subdirectories or visit the online playground and select "JSON output".
+
+## Handling errors
+
+Errors occur in all pieces of code, including your packages. As a part of the sandbox we build to run your packages in, we have built in some error handling.
```

Comment:
Typo, "built". But might be better to skip the sentence about entirely since the error system is not really coupled to exclusively the playground.

Reply:
Hmm, I don't really understand what you are aiming for now. We do build a sandbox environment for the packages to run, right? It isn't specific to the playground, we use Wasmer in both native and web-targeted builds. With "sandbox", I am referring to the Wasmer runtime. Do you think we should skip the information that errors will be caught by our sandbox? Or is it better to rephrase it?